run_test.sh

```diff
@@ -165,8 +165,6 @@ run_status_with_expected_output() {
 generalize_expected_output() {
     local output_log=$1
 
-    # "depmod..." lines can have multiple points. Replace them, to be able to compare
-    sed -i 's/\([^.]\)\.\.\.\.*$/\1.../' ${output_log}
```

Comment:
I have a different suggestion here that could fix the Ubuntu VM issue and in general be more robust to random extra newlines. What if we just remove all empty lines and then remove the extra newlines in the expected output?

```
# Remove lines containing only whitespace to normalize newline differences
sed -i '/^\s*$/d' ${output_log}
```

Reply:
We could, but then I don't think we will match the intended result in a merge request (for example) to... change the output.

Reply:
I feel like there is some reason the original author created a `dkms_test` module and added these tests. IMO if we can keep the test working that would be better, but as primarily a user of DKMS and only recently reviewing the code I don't think I would be able to say I have any strong opinions to keeping or removing the test. I'll leave that judgement best to @scaronni.